### DISCUSSION TITLE
Not showing page number at specified page numbers

### DISCUSSION THREAD

**Discussion by psads-git** [typst/typst#4069] [Q&A] [answered]
I would like to create a show rule that does not show the page number when page number is equal to 1.

Could you please help me? Thanks!

**Comment by EpicEricEE** [comment, marked as answer]
I don't think you can do this with a show-rule, but you certainly can with a set-rule.

When you're inside a context block, you can check the current page number and only show it if it's bigger than one:

```typ
#set page(
  footer: context align(center, {
    if counter(page).get().first() > 1 {
      let page-numbering = page.numbering
      if page-numbering == none { page-numbering = "1" }
      counter(page).display(page-numbering)
    }
  })
)
```

**Comment by psads-git** [comment]
Thanks a lot, @EpicEricEE: that works perfectly!
